pr_reminder.rb

```diff
@@ -3,6 +3,50 @@
 require 'json'
 require 'ostruct'
 
+class GitHubAPI
+  @@api_token_file = File.expand_path(File.dirname(__FILE__)) + "/api_token"
+  @@api_token = nil
+
+  def self.token
+    if !@@api_token
+      if File.exists?(@@api_token_file)
```

Comment:
Can you also allow setting it with ENV variable? it will be more useful way for us, as we run it on jenkins.